packages/@webex/plugin-meetings/test/unit/spec/meeting/voicea-meeting.ts

```diff
@@ -0,0 +1,266 @@
+import {
```

Comment:
* Base PR - https://github.com/webex/webex-js-sdk/pull/3336
* Voicea for Guest users & stopTranscription - https://github.com/webex/webex-js-sdk/pull/3405
* Voicea UT in meeting/index.ts - https://github.com/Shreyas281299/webex-js-sdk/pull/12
* Voicea Utils UT - https://github.com/webex/webex-js-sdk/pull/3452